src/Uno.UI/UI/Xaml/Media/Animation/Animators/GPUFloatValueAnimator.iOSmacOS.cs

```diff
@@ -126,12 +130,17 @@ public void Start()
 			}
 
 			InitializeCoreAnimation();
+			if (CoreWindow.GetForCurrentThread() is { } coreWindow)
+			{
+				coreWindow.VisibilityChanged += OnCoreWindowVisibilityChanged;
```

Comment:
I wonder if we should be maintaining a static weak list of active animators and only register once to the VisibilityChanged event. It is relatively expensive to add an event handler to a multicast delegate.